### DISCUSSION TITLE
How to use mtex merging two or more EBSD inverse pole figures

### DISCUSSION THREAD

**Discussion by kevindurantnets** [mtex-toolbox/mtex#2121] [Ask Anything] [answered]
Hi, 
    I want to know how to merge two or more EBSD pole figures by using mtex. I scanned different areas of the same material on two separate occasions and printed two pole figures , but I don't konw how to merge these two pole figures to analyse textures. 
    Thanks a lot.

**Comment by lfgmorales** [comment, marked as answer]
Hi

You have to concatenate the maps

ebsd_t=[ebsd_1, ebsd_2]

Will do

Cheers

Luiz


Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 19 Apr 2024, at 11:29, kevindurantnets ***@***.***> wrote:

﻿

Hi,
I want to know how to merge two or more EBSD pole figures by using mtex. I scanned different areas of the same material on two separate occasions and printed two pole figures , but I don't konw how to merge these two pole figures to analyse textures.
Thanks a lot.

—
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/2121>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5IN4DHTFMBICSF4ZREN3Y6DPV5AVCNFSM6AAAAABGO2QZ6SVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZWGUZDOOJRGM>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/2121", "url": "https://github.com/mtex-toolbox/mtex/discussions/2121", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Comment by kevindurantnets** [comment]
Thanks a lot for your help. but when I want to merge EBSD inverse pole figures by using the code as shown below, it reminds me that "The number of values should match the number of ebsd data", how to solve this problem?
ebsd_t=[ebsd,ebsd2]
plot(ebsd_t,ipfkey.orientation2color(ebsd2.orientations))

**Comment by lfgmorales** [comment]
Try substituting ebsd2.orientations by ebsd_t.orientations

Cheers

L
Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 19 Apr 2024, at 12:00, kevindurantnets ***@***.***> wrote:

﻿

Thanks a lot for your help. but when I want to merge EBSD inverse pole figures by using the code as shown below, it reminds me that "The number of values should match the number of ebsd data", how to solve this problem?
ebsd_t=[ebsd,ebsd2]
plot(ebsd_t,ipfkey.orientation2color(ebsd2.orientations))

—
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/2121#discussioncomment-9164865>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5IN56FHXSIRAKD2ZYNTDY6DTMFAVCNFSM6AAAAABGO2QZ6SVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TCNRUHA3DK>.
You are receiving this because you commented.Message ID: ***@***.***>

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/2121#discussioncomment-9164865", "url": "https://github.com/mtex-toolbox/mtex/discussions/2121#discussioncomment-9164865", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]
